tests/test_TFUtil.py

```diff
@@ -469,6 +469,43 @@ def test_Data_copy_compatible_to_batch_feature_is_dynamic():
   assert_equal(t.get_size_dim_tag(0), dec.get_time_dim_tag())
 
 
+def test_Data_get_common_data_modified_sources():
+  from TFNetworkLayer import WindowLayer, InternalLayer, ReduceLayer, SliceLayer
+  from TFNetwork import TFNetwork, ExternData
```

Comment:
All tests in `test_TFUtil.py` should just depend on `TFUtil`, nothing else. Please simplify this test case. Or move it to `test_TFNetworkLayer.py` or so. Or better both.

Reply:
Done